geo/aws.go

```diff
@@ -0,0 +1,121 @@
+package geo
+
+import (
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"os"
+)
+
+const (
+	AWSGeoUrl  = "https://ip-ranges.amazonaws.com/ip-ranges.json"
+	AWSGeoFile = "ip-ranges.json"
+)
+
+type AWSProvider struct {
+}
+
+type AWSGeoInfo struct {
+	SyncToken  string      `json:"syncToken"`
+	CreateData string      `json:"createDate"`
+	Prefixes   []AWSIPInfo `json:"prefixes"`
+}
+
+type AWSIPInfo struct {
+	IPPrefix           string `json:"ip_prefix"`
+	Region             string `json:"region"`
+	Service            string `json:"service"`
+	NetworkBorderGroup string `json:"network_border_group"`
+}
+
+var AWSRegionMapping = map[string]string{
+	"us-east-1":      "US",
+	"us-east-2":      "US",
+	"us-west-1":      "US",
+	"us-west-2":      "US",
+	"af-south-1":     "ZA",
+	"ap-east-1":      "HK",
+	"ap-south-1":     "IN",
+	"ap-northeast-1": "JP",
+	"ap-northeast-2": "KR",
+	"ap-southeast-1": "SG",
+	"ap-southeast-2": "AU",
+	"ca-central-1":   "CA",
+	"eu-central-1":   "DE",
+	"eu-west-1":      "IE",
+	"eu-west-2":      "EN",
```

Comment:
Where did you get `EN`? It should be `GB`: https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes#UNI2

Reply:
Also please double check other codes to make sure there are no mistakes